javascript/javascript2/week1/problemSolvingCardio.js

```diff
@@ -0,0 +1,25 @@
+//First excercise: Find the shortest word.
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function findTheShortestWord(array) {
```

Comment:
The only thing which i wanted to mentioned is in your solution where you sorting the array, in the first place is always the shortest words comes, and your result is just one element with the 0 index, but what if you will have a few words with the same length and they will have index 0,1, and 2? And how can you make that happen?

Here is my suggestion:

`function findTheShortestWord(array) {
  const sorted = array.sort(function (a, b) {
    return a.length - b.length;
  });

  const shortest = sorted.filter((word) => word.length === sorted[0].length);
  console.log({ shortest });
}

findTheShortestWord(danishWords);`